src/components/ContributionTable.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <v-col>
+    <v-card cols="12" min-width="320">
+      <v-card-item>
+        <v-card-title><slot /></v-card-title>
+      </v-card-item>
+      <v-card-text>
+        <v-table  class="contributionTable">
+          <tbody>
+            <tr v-for="contribution in content" :key="contribution.dateModified">
+              <td>{{ convertUTCtoDate(contribution.dateModified) }}</td>
+              <td v-if="this.global">
+                <a :href=generateProfileURL(contribution.user) target="_blank">{{ contribution.user }}</a></td>
```

Comment:
Here's where I make use of the global prop flag.  I was looking for a way to distinguish between a table component that would display the current user's activity and one that would display the latest global activity; in the case of the global table, I wanted to include the username of whomever made the contribution.

I thought about doing something like  `<td v-if="contribution.user != currentUser">` but realized that that would produce undesired results if the current user was responsible for an item on the "latest activity" table.